compose.yaml

```diff
@@ -0,0 +1,22 @@
+services:
+  db:
+    hostname: postgres
+    image: "postgres"
+    volumes:
+      - data:/etc/data
+    ports:
+      - "5432:5432"
+    shm_size: 128mb
+    environment:
+      POSTGRES_PASSWORD: postgres
+  backend:
+    image: backend
+    volumes:
+      - data:/etc/data
+    build: .
+    ports:
+      - "80:80"
+    env_file: ../reporting-service/defaults.env
```

Comment:
maybe this should be changed to the correct file

Reply:
Right, I was also thinking of using the environment key instead of env_file, list all the vars and load them from a .env file.
Alongside this I would also add an example.env file that a user can modify for their needs, that would contain all necessary keys